src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -405,10 +404,6 @@ public void hoverClickAndViewPhotoOnTableCell(int questionBodyIndex, int tableRo
         
         waitForElementPresence(By.cssSelector(".popover-content"));
         
-        WebElement popoverContent = browser.driver.findElement(By.className("popover-content"));
-        
-        waitForElementVisibility(popoverContent);
-        
         jsExecutor.executeScript("document.getElementsByClassName('popover-content')[0]" +
```

Comment:
will removing the `waitForElementVisibility` call break the `document.getElementsByClassName` part?